applications/ConstitutiveLawsApplication/tests/cpp_tests/test_tangent_tensor_numerical_derivation_utility.cpp

```diff
@@ -107,7 +107,7 @@ void SettingBasicCase(
  */
 void Create3DGeometryHexahedra(
     ModelPart& rThisModelPart,
-    const std::string ElementName = "SmallDisplacementElement3D8N"
+    const std::string& ElementName = "SmallDisplacementElement3D8N"
```

Comment:
`std::string_view` might be better suited, but I have never used it

Reply:
I am not trying this level of refactor in this PR. See I don't even change the name of the variables